gax-java/gax-grpc/src/test/java/com/google/api/gax/grpc/TimeoutTest.java

```diff
@@ -55,22 +55,18 @@
 import io.grpc.MethodDescriptor.MethodType;
 import java.io.IOException;
 import java.util.concurrent.TimeUnit;
-import org.junit.BeforeClass;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.ArgumentCaptor;
 import org.mockito.ArgumentMatchers;
 import org.mockito.Mock;
 import org.mockito.Mockito;
-import org.mockito.junit.MockitoJUnit;
-import org.mockito.junit.MockitoRule;
-import org.mockito.quality.Strictness;
+import org.mockito.junit.jupiter.MockitoExtension;
 import org.threeten.bp.Duration;
 
-@RunWith(JUnit4.class)
-public class TimeoutTest {
+@ExtendWith(MockitoExtension.class)
```

Comment:
Why do we need this?

Reply:
Because there is a `@Rule` for Mockito being removed below.